spring-core/pom.xml

```diff
@@ -13,7 +13,7 @@
   <description>Togglz - Spring Framework integration</description>
 
   <properties>
-    <spring.version>3.0.7.RELEASE</spring.version>
+    <spring.version>3.1.4.RELEASE</spring.version>
```

Comment:
I'm basically fine with increasing the version to 3.1.4.Final. I just did some research and it looks like even Spring 3.2.x will be EOLed by the end of this year. So moving to 3.1.x is fine.